### ISSUE TITLE
When generating axioms report out of range references as an error in the transformation rule

### ISSUE THREAD

**Issue by csnyulas** [protegeproject/cellfie-plugin#32]
If a transformation rule references a cell that is out of the spreadsheet's range - the reference being either within the rule itself (such as @F100 in a spreadsheet that has less then 6 columns and/or less then 100 rows), or through its range of application (for example the specified end row/end column in the transformation rule is bigger than the last row/column in the spreadsheet) - and the user clicks the "Generate axiom" button, an "Out of Range" error should be reported/logged.
Basically, the row and column of each reference should be checked against the last row and last column  of the spreadsheet (which is the last cell to contain some value).

While in case of out of range cell references the desired action is to report an error (as described above), empty cells **within** the valid range of the spreadsheet should be handled gracefully as containing an empty string (see related item #31).

**Comment by johardi** [comment]
I'd prefer to have a silent warning instead of triggering an error. An error will put a stop to the application which can be annoying for a less-critical mistake.
